test/powershell/Modules/Microsoft.PowerShell.Utility/Format-List.Tests.ps1

```diff
@@ -183,3 +210,17 @@ dbda : KM
         $actual | Should -BeExactly $expected
     }
 }
+
+Describe 'Format-List color tests' {
+    BeforeAll {
+        $PSDefaultParameterValues.Add('It:Skip', (-not $EnabledExperimentalFeatures.Contains('PSAnsiRendering')))
+    }
+
+    AfterAll {
+        $PSDefaultParameterValues.Remove('It:Skip')
+    }
+
+    It 'Property names should use FormatAccent' {
+        ([pscustomobject]@{foo = 1} | Format-List | Out-String).Trim() | Should -BeExactly "$($PSStyle.Formatting.FormatAccent)foo : $($PSStyle.Reset)1"
```

Comment:
hmmmm, I wonder if we should extend `out-string` to include a new parameter for psstyle

Reply:
I've thought about `Out-String -OutputRendering` type parameter, but if there's interest, that should be an issue outside of this PR